app/javascript/components/pages/CollectionShow.js

```diff
@@ -1,8 +1,49 @@
 import React from 'react'
+import { NavLink, useParams } from 'react-router-dom'
+import {Card, CardBody, Button, CardTitle, CardSubtitle} from "reactstrap"
+
+const CollectionShow = ({collections}) => {
+  let {id} = useParams()
+  const currentCollection = collections?.find((collection) => collection.id === +id)
 
-const CollectionShow = () => {
   return (
-    <div>CollectionShow</div>
+    <>
+      <h1>Your Collectible</h1>
+      {currentCollection &&
+        <Card
+          style={{
+            width: '18rem'
+          }}
+        >
+          <img
+            alt={currentCollection.name}
+            src={currentCollection.image}
+          />
+          <CardBody>
+            <CardTitle tag="h5">
+              {currentCollection.name}
+            </CardTitle>
+            <CardSubtitle>
+              Category: {currentCollection.category}, Description:
+            {currentCollection.description}
```

Comment:
Did you want this in separate lines?

Reply:
I will put it on the same line as the other.